deel-2-flexbox-layout/opdracht-2/styles/styles.css

```diff
@@ -1 +1,32 @@
-/*Schrijf hier jouw CSS*/
\ No newline at end of file
+/*Schrijf hier jouw CSS*/
+
+body {
+    display: flex;
+    height: 100vh;
+}
+
+.col-1 {
+    display: flex;
+    flex-basis: 500px;
+}
+
+.col-2 {
+    display: flex;
+    flex-direction: column;
+    flex-basis: 2000px;
+}
+
+header {
+    display: flex;
+    flex-basis: 150px;
+}
+
+article {
+    display: flex;
+    height: 900px;
+}
```

Comment:
je article staat nu vanuit de body in een column container, als je de loremipsum tekst aanzet zie je nu dat de tekst in het artikel niet op de juiste manier staat. Je dient daarom nog een container van de main te maken, en die op row te zetten. Op die manier wordt de tekst in het artikel netjes van links naar rechts en van boven naar beneden in de juiste leesrichting weergegeven.